cse/indexing/DeltaPostingListIndex.py

```diff
@@ -14,7 +14,7 @@ class DeltaPostingListIndex(object):
     __numCommentIds = 0
 
 
-    def __init__(self, cidSize=35): # import sys; cid = "14d2c537-d2ed-4e36-bf3d-a26f62c02370"; assert(sys.getsizeof(cid) == 85)
+    def __init__(self, cidSize=42): # import sys; cid = "14d2c537-d2ed-4e36-bf3d-a26f62c02370"; assert(sys.getsizeof(cid) == 85)
```

Comment:
I count 36 characters

Reply:
bytes used to store the string in memory, not chars
but I adopted the size to match more closely to the needed size on disk (via experiments)